connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -718,7 +718,6 @@ object SparkSession extends Logging {
      *
      * This will always return a newly created session.
      */
-    @deprecated(message = "Please use create() instead.", since = "3.5.0")
```

Comment:
This is connect library which is relatively fine to deprecate and remove unlike other API that has been years. let's just deprecate this.

Reply:
resonable